service-impl/palisade-service-impl/simple-palisade-service/src/main/java/uk/gov/gchq/palisade/service/impl/SimplePalisadeService.java

```diff
@@ -244,21 +259,55 @@ private MultiPolicy getPolicy(final RegisterDataRequest request, final Completab
                 }).join();
     }
 
-    private void audit(final RegisterDataRequest request, final User user, final MultiPolicy multiPolicy, final AuditType auditType) {
+    private void auditProcessingStarted(final RegisterDataRequest request, final User user, final MultiPolicy multiPolicy) {
         for (final Entry<LeafResource, Policy> entry : multiPolicy.getPolicies().entrySet()) {
-            final AuditRequest auditRequest =
-                    new AuditRequest()
+            final AuditRequestProcessingStarted auditRequestProcessingStarted =
+                    new AuditRequestProcessingStarted()
                             .resource(entry.getKey())
-                            .auditType(auditType)
                             .user(user)
                             .context(request.getContext())
                             .howItWasProcessed(entry.getValue().getMessage());
-            LOGGER.debug("Auditing: {}", auditRequest);
-            auditService.audit(auditRequest);
+            LOGGER.debug("Auditing: {}", auditRequestProcessingStarted);
+            auditService.audit(auditRequestProcessingStarted);
         }
     }
 
-    private void cache(final RegisterDataRequest request, final User user, final RequestId requestId, final MultiPolicy multiPolicy, final int resCount) {
+    private void auditProcessingComplete(final RegisterDataRequest request, final User user, final RequestId requestId) {
```

Comment:
The Palisade service is unable to state that the processing completed since it is the data service that supplies the data and errors might occur serving the data or the client might not actually back the request to get the data